tests/components/yandex_transport/test_yandex_transport_sensor.py

```diff
@@ -0,0 +1,138 @@
+"""Tests for the yandex transport platform."""
+
+from json import loads
```

Comment:
Please `import json`. It's short enough and that makes it clear what we're loading when we call `json.loads`.